qiita_db/study.py

```diff
@@ -789,6 +802,87 @@ def _portals(self):
             TRN.add(sql, [self._id])
             return TRN.execute_fetchflatten()
 
+    @property
+    def ebi_study_accession(self):
+        """The EBI study accession for this study
+
+        Returns
+        -------
+        str
+            The study EBI accession
+        """
+        with TRN:
+            sql = """SELECT ebi_study_accession
+                     FROM qiita.{0}
+                     WHERE study_id = %s""".format(self._table)
+            TRN.add(sql, [self._id])
+            return TRN.execute_fetchlast()
+
+    @ebi_study_accession.setter
+    def ebi_study_accession(self, value):
+        """Sets the study's EBI study accession
+
+        Parameters
+        ----------
+        value : str
+            The new EBI study accession
+
+        Raises
+        ------
+        QiitDBError
+            If the study already has an EBI study accession
+        """
+        with TRN:
+            if self.ebi_study_accession is not None:
+                raise QiitaDBError(
+                    "Study %s already has an EBI study accession"
+                    % self.id)
+            sql = """UPDATE qiita.{}
+                     SET ebi_study_accession = %s
+                     WHERE study_id = %s""".format(self._table)
+            TRN.add(sql, [value, self.id])
+            TRN.execute()
+
+    @property
+    def ebi_submission_status(self):
+        """The EBI submission status of this study
+
+        Returns
+        -------
+        str
+            The study EBI submission status
+        """
+        with TRN:
+            sql = """SELECT ebi_submission_status
+                     FROM qiita.{0}
+                     WHERE study_id = %s""".format(self._table)
+            TRN.add(sql, [self.id])
+            return TRN.execute_fetchlast()
+
+    @ebi_submission_status.setter
+    def ebi_submission_status(self, value):
+        """Sets the study's EBI submission status
+
+        Parameters
+        ----------
+        value : str {%s}
+            The new EBI submission status
+
+        Raises
+        ------
+        ValueError
+            If the status is not known
+        """.format(', '.join(self._valid_ebi_status))
```

Comment:
http://stackoverflow.com/questions/10307696/how-to-put-a-variable-into-python-docstring